tests/run.sh

```diff
@@ -4,11 +4,13 @@ set -e # Exit on error
 
 SCRIPT_PATH="`dirname \"$0\"`"
 
+NUMCPUS=`grep -c ^processor /proc/cpuinfo`
```

Comment:
Is `$(nproc)` more portable? It's part of coreutils, as far as I know.

Reply:
Thanks @embeddedt - `nproc` is the way to go. WDYT about how I'm installing parallel in the "Install SDL" step. Should I rename it to be less SDL specific, run create a second install step, or leave as-is?

Reply:
Personally I don't mind if it's left as "Install SDL", but it might be clearer to rename it to "Install dependencies".